tests/regionizers/test_h3_regionizer.py

```diff
@@ -0,0 +1,30 @@
+"""Tests for H3Regionizer."""
+
+import osmnx as ox
+import pytest
+
+from srai.regionizers.h3_regionizer import H3Regionizer
+
+
+@pytest.mark.parametrize(  # type: ignore
+    "geoname,resolution,buffer",
+    [
+        ("Wroclaw, Poland", 7, True),
+        ("Tokyo, Japan", 7, False),
+        ("London, England", 7, True),
+    ],
+)
+def test_transform(geoname: str, resolution: int, buffer: bool) -> None:
+    """
+    Tests transform.
+
+    Args:
+        geoname (str): name of the place
+        resolution (int): resolution for H3 Regionizer
+        buffer (bool): whether to buffer the H3 cells to fill the borders
+
+    """
+    gdf = ox.geocode_to_gdf(geoname)
+    gdf_h3 = H3Regionizer(resolution, buffer).transform(gdf)
+
+    assert len(gdf_h3) > 0
```

Comment:
Maybe another test for asserting that buffered transform covers whole geometry: 
- merge using `_merge_disjointed_gdf_geometries`
- make the difference between the original GDF and buffered hexagonal regions
- make sure that it's empty (`is_empty`)

Reply:
Yep, need to add some more tests